### PR TITLE
add .md file/entry for Pandas dataframe tail

### LINKED ISSUE #4523: [Term Entry] Python:Pandas dataframe tail
### Reason/inspiration (optional)

"We would like a new term entry in the `dataframe` concept for [Pandas](https://www.codecademy.com/resources/docs/pandas): tail. The entry should go in a new file under `docs/content/pandas/concepts/dataframe/terms/tail/tail.md`.

The entry should describe the term and how it is used. Also include: 
- An "Example" section that explains to readers how to use the trace function to find the trace of a square matrix.
- Should contain information for functions and syntax.
- Should have a Python codebyte with a working code snippet showcasing this function.
Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue."

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by sethaferd**
### Description

Added .md file/entry for Pandas dataframe tail.

### Issue Solved

Closes #4523 

### Type of Change

- Adding a new entry

### Checklist

- [ x ] All writings are my own.
- [ x ] My entry follows the Codecademy Docs style guide.
- [ x ] My changes generate no new warnings.
- [ x ] I have performed a self-review of my own writing and code.
- [ x ] I have checked my entry and corrected any misspellings.
- [ x ] I have made corresponding changes to the documentation if needed.
- [ x ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ x ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ x ] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by mamtawardhani** [comment]
Hey @sethaferd, seems like the path of entry is incorrect as the `tail.md` file should be inside a folder named `tail`, could you please update that? This is how your final path should look like: `docs/content/pandas/concepts/dataframe/terms/tail/tail.md`.

Let me know once you update it 😃

File: content/pandas/concepts/dataframe/terms/tail/tail.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+Title: '.copy()'
```
Comment (mamtawardhani) [inline content/pandas/concepts/dataframe/terms/tail/tail.md:2]:
```suggestion
Title: '.tail()'
```

Comment (mamtawardhani) [reply content/pandas/concepts/dataframe/terms/tail/tail.md:2]: The entry is about the `.tail()` method, seems like a silly mistake 🥹

File: content/pandas/concepts/dataframe/terms/tail/tail.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  - 'Web Development'
+Tags:
+  - 'Data Structures'
+  - 'Pandas'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/data-science'
+---
+
+In Pandas, **`.tail()`** is a method that returns the last n rows of a DataFrame. By default, it returns the last 5 rows, but the number of rows can be adjusted by passing an integer argument to the method.
```
Comment (mamtawardhani) [inline content/pandas/concepts/dataframe/terms/tail/tail.md:16]:
```suggestion
In Pandas, **`.tail()`** is a method that returns the last `n` rows of a [DataFrame](https://www.codecademy.com/resources/docs/pandas/dataframe). By default, it returns the last 5 rows, but the number of rows can be adjusted by passing an integer argument to the method.
```

File: content/pandas/concepts/dataframe/terms/tail/tail.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+## Syntax
+
+```pseudo
+df_tail = df.tail(n=5)
```
Comment (mamtawardhani) [inline content/pandas/concepts/dataframe/terms/tail/tail.md:21]:
```suggestion
DataFrame.tail(n=5)
```

File: content/pandas/concepts/dataframe/terms/tail/tail.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+```
+
+The parameter `n` specifies the number of rows to return. The default value is `n=5`.
```
Comment (mamtawardhani) [inline content/pandas/concepts/dataframe/terms/tail/tail.md:24]: Describing the parameters should be done in the following format:

- `DataFrame`: The Pandas data structure on which the method is called.
- `n`: Number of rows to return from the end of the DataFrame. The default value is 5.

(You can modify the framing of the sentences, this is just for example)

File: content/pandas/concepts/dataframe/terms/tail/tail.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+The output of the code above will be:
+
+```shell
+   A   B
+3  4  10
+4  5  11
+5  6  12
+```
+
+By using the `.tail()` method, the end of a DataFrame can be previewed, which can be useful for inspecting the most recent data in a dataset.
```
Comment (mamtawardhani) [inline content/pandas/concepts/dataframe/terms/tail/tail.md:50]:
```suggestion
By using the `.tail()` method, the end of a DataFrame can be previewed, which is useful for inspecting the most recent data in a dataset.
```

Comment (mamtawardhani) [reply content/pandas/concepts/dataframe/terms/tail/tail.md:50]: At the end, please also add a running codebyte example using the ##Codebyte Example block, please note that the codebyte example is not accompanied by an output block.

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @sethaferd, thank you for contributing to the Codecademy Docs, the entry is nicely written!

I've requested some minor changes, could you please review and modify them at your earliest convenience? Thank you! 😃

**Review by mamtawardhani — APPROVED** [review]
Thank you for the contribution @sethaferd! 😃 

The entry looks good for a second review 🚀

**Review by Sriparno08 — APPROVED** [review]
Looks good, @sethaferd! 👍

**Pull request merged**